packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -38,12 +35,22 @@ class JavaObject with Copyable {
           instanceManager: instanceManager,
         );
 
+  static InstanceManager? _globalInstanceManager;
```

Comment:
The less error-prone way of doing lazy initialization is:

```
static InstanceManager globalInstanceManager = _initInstanceManager();

static final InstanceManager _initInstanceManager() {
  WidgetsFlutterBinding.ensureInitialized();
  // Clears the native `InstanceManager` on initial use of the Dart one.
  InstanceManagerHostApi().clear();
  _globalInstanceManager = InstanceManager(
    onWeakReferenceRemoved: (int identifier) {
      JavaObjectHostApiImpl().dispose(identifier);
    },
  );
}
```

(For non-statics, you can do the same thing but with `late final` to make it lazy.)

Reply:
cc @camsim99 FYI when updating the `InstanceManager`.

Reply:
Updated example

```dart
/// Global instance of [InstanceManager].
static final InstanceManager globalInstanceManager = _initInstanceManager();

static InstanceManager _initInstanceManager() {
  WidgetsFlutterBinding.ensureInitialized();
  // Clears the native `InstanceManager` on initial use of the Dart one.
  InstanceManagerHostApi().clear();
  return InstanceManager(
    onWeakReferenceRemoved: (int identifier) {
      JavaObjectHostApiImpl().dispose(identifier);
     },
  );
}
```

Reply:
Oops, I'll fix my copy to avoid confusion. Sorry about that.